ocaml/xapi/xapi_host.ml

```diff
@@ -109,17 +109,17 @@ let assert_safe_to_reenable ~__context ~self ~user_request =
       )
       unplugged_pbds ;
     let pifs = Db.Host.get_PIFs ~__context ~self in
-    let unplugged_pifs =
+    let unpluggable_pifs =
```

Comment:
"unpluggable" to me sounds like there are plugged and can be unplugged, but what you mean is the other way around...